odoo/modules/module.py

```diff
@@ -452,11 +452,14 @@ def adapt_version(version: str) -> str:
     version_str_parts = version.split('.')
     if not (2 <= len(version_str_parts) <= 5):
         raise ValueError(f"Invalid version {version!r}, must have between 2 and 5 parts")
+    serie = release.major_version
```

Comment:
Couldn't we use `release.serie` directly instead of defining a `serie`variable.
As we have this line in release.py:
`series = serie = major_version = '.'.join(str(s) for s in version_info[:2])`

Reply:
It is just because it is used multiple times in that function.